.travis.yml

```diff
@@ -43,7 +42,6 @@ script:
 
 after_script:
   - wget https://scrutinizer-ci.com/ocular.phar
-  - if [ $(phpenv version-name) = "5.5" ] && [ "$REMOVE_XDEBUG" = "0" ]; then php ocular.phar code-coverage:upload --format=php-clover tests/clover.xml --revision=$TRAVIS_COMMIT; fi
```

Comment:
Yes it's two lines after the line of the diff you linked. I now require `PHP >= 5.6` in `composer.json`